CHANGELOG.md

```diff
@@ -70,6 +73,34 @@
 - [0.2.0](#020)
 - [0.1.0](#010)
 
+## [v0.59.1]
+
+> Release date: 2024/09/12
+- fix: Prevent unset records to be filled with empty tables: {}
+  for deck files.
```

Comment:
Can we be a little more descriptive with this entry? I'm having hard time understanding this one 🤔